install_config/build_defaults_overrides.adoc

```diff
@@ -0,0 +1,109 @@
+= Configuring Global Build Defaults and Overrides
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Developers can define settings in specific build configurations within their
+projects, such as
+link:../dev_guide/builds.html#using-a-proxy-for-git-cloning[configuring a proxy
+for Git cloning]. Rather than requiring developers to define certain settings in
+each of their build configurations, cluster administrators can use admission
+control plug-ins to configure global build defaults and overrides that
+automatically populate these settings into any build configuration.
```

Comment:
s/populate these settings into any build configuration./use these settings in any build./

The settings from the plugin config don't get set on the build config or builds, they actually only get used during the build. That way, at any time you can change your global config and re-run builds from those builds/build configs and get the new settings.